File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q26.js
import { question } from "readline-sync";
function main(){
    //Input
    const value_day = parseInt(question("Enter a value in days: "))

    //Processing
    const value_week = Math.floor(value_day / 7)
    const value_remainder_days = value_day % 7

    //Output
    console.log(`The value of ${value_day} days converted to weeks and days is equal to: ${value_week} week(s) and ${value_remainder_days} day(s).`)

        
}

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q26.py
def main():
    #Input
    value_day = int(input("Enter a value in days: "))

    #Processing
    value_week = round(value_day / 7) 
    value_day_remainder = value_day % 7
    
    #Output
    print("The entered value of {value_day} days converted to week and day is equal to: {value_week} weeks and {value_day_remainder}")

main()